src/Files.App.Controls/Files.App.Controls.csproj

```diff
@@ -0,0 +1,19 @@
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <UseWinUI>true</UseWinUI>
+    <SupportedOSPlatformVersion>10.0.19041.0</SupportedOSPlatformVersion>
+    <Platforms>x64;x86;ARM64</Platforms>
+  </PropertyGroup>
```

Comment:
@0x5bfa should we use shared properties for the new projects?

Reply:
FWIW, The Controls project should be compatible with the main app, so if linking keeps it matching, that sounds right.

The UITests app project should also always match the main app, so it can be a reliable testing app

Reply:
Yes Directory.props to avoid mismatches.
Do you like us to introduce it here?

Reply:
I think starting with a new project is a good idea, over time we can adopt for the older projects as well.

Reply:
I will trust you both know what you are talking about, and @0x5bfa can do it and this can be merged

Reply:
Done.
Ideally, we have a directory ‘platform’ or ‘windows’ for ‘Files.App.*’ projects to have a shared properties file for Windows-specific properties such as ‘TargetPlatformMinVersion’ and ‘SupportedOSPlatformVersion’.

Reply:
Btw, we can delete some properties which are already shared in all existing project files but I don’t update to keep things small